### PR TITLE
Bjin cloudflare/[china network] Update with China Express

### PR DESCRIPTION
**Pull request by bjin-cloudflare**
### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/docs/china-network/index.mdx [hunk 1]
```diff
@@ -21,10 +21,11 @@ The Cloudflare China Network provides:
 * An unified experience for managing network traffic and security posture. You can manage all configurations on the same dashboard.
 * The same customer support capabilities as Cloudflare’s global network. You may also have access to premium service and local language support.
 * [In-China Authoritative DNS and in-China nameservers](/china-network/concepts/china-dns/) to improve the Time to First Byte (TTFB) performance.
+* [China Express](/china-network/concepts/china-express) is a suite of connectivity, performance offerings and professional services designed to simplify your global assets' deployment in China.
 
 ## Availability
 
-The Cloudflare China Network is available as a separate subscription for customers on an Enterprise plan.
+The Cloudflare China Network is available as a separate subscription for customers on an [Enterprise plan](https://www.cloudflare.com/plans/enterprise/).
```
Comment (bjin-cloudflare) [inline src/content/docs/china-network/index.mdx:27]: Add link pointing to the Enterprise Plan

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 2]
```diff
@@ -35,24 +34,23 @@ Default vs. In-China Nameserver
 
 After you [enable the Cloudflare China Network service](/china-network/get-started/), do the following:
 
-1. Contact your account team to enable the feature. Currently, you cannot enable China Authoritative DNS in the Cloudflare dashboard.
```
Comment (bjin-cloudflare) [inline src/content/docs/china-network/concepts/china-dns.mdx:38]: Reviewed and suggested by [crwaters16](https://github.com/crwaters16)

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 3]
```diff
@@ -35,24 +34,23 @@ Default vs. In-China Nameserver
...
+1. You will need to reach out to your Cloudflare sales team to enable the feature. Currently the feature can't be enabled from the Cloudflare dashboard.
 
-    The Cloudflare China Network currently supports domains using a [full setup](/dns/zone-setups/full-setup/) or a [partial setup](/dns/zone-setups/partial-setup/).
```
Comment (bjin-cloudflare) [inline src/content/docs/china-network/concepts/china-dns.mdx:40]: Reviewed and suggested by [crwaters16](https://github.com/crwaters16)

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 4]
```diff
@@ -35,24 +34,23 @@ Default vs. In-China Nameserver
...
+    The current China Network supports both a [full setup](/dns/zone-setups/full-setup/) and a [partial setup](/dns/zone-setups/partial-setup/).
 2. Update your domain registrar with the assigned in-China nameservers.
 
-   * Full setup: These nameservers are displayed in the Cloudflare dashboard.
+    a. Full setup: These name servers are displayed in the Cloudflare dashboard.
 
-    * Partial setup: CNAME to \<host_name\>.cdn.cloudflarecn.net.
+    b. Partial setup: CNAME to \<host_name\>.cdn.cloudflarecn.net.
```
Comment (bjin-cloudflare) [inline src/content/docs/china-network/concepts/china-dns.mdx]: [crwaters16](https://github.com/crwaters16)

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 5]
```diff
@@ -35,24 +34,23 @@ Default vs. In-China Nameserver
...
 After you [enable the Cloudflare China Network service](/china-network/get-started/), do the following:
 
-1. Contact your account team to enable the feature. Currently, you cannot enable China Authoritative DNS in the Cloudflare dashboard.
+1. You will need to reach out to your Cloudflare sales team to enable the feature. Currently the feature can't be enabled from the Cloudflare dashboard.
 
-    The Cloudflare China Network currently supports domains using a [full setup](/dns/zone-setups/full-setup/) or a [partial setup](/dns/zone-setups/partial-setup/).
+    The current China Network supports both a [full setup](/dns/zone-setups/full-setup/) and a [partial setup](/dns/zone-setups/partial-setup/).
 2. Update your domain registrar with the assigned in-China nameservers.
 
-   * Full setup: These nameservers are displayed in the Cloudflare dashboard.
```
Comment (bjin-cloudflare) [inline src/content/docs/china-network/concepts/china-dns.mdx:43]: Reviewed and suggested by [crwaters16](https://github.com/crwaters16)

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 6]
```diff
@@ -35,24 +34,23 @@ Default vs. In-China Nameserver
...
+    a. Full setup: These name servers are displayed in the Cloudflare dashboard.
 
-    * Partial setup: CNAME to \<host_name\>.cdn.cloudflarecn.net.
```
Comment (bjin-cloudflare) [inline src/content/docs/china-network/concepts/china-dns.mdx:45]: Reviewed and suggested by [crwaters16](https://github.com/crwaters16)

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 7]
```diff
@@ -35,24 +34,23 @@ Default vs. In-China Nameserver
...
+    b. Partial setup: CNAME to \<host_name\>.cdn.cloudflarecn.net.
 
     For example:
-            You have a zone named samplezone.cn. In this zone, you have two DNS records: www and media, pointing to two different origins. Your Authoritative DNS server needs to set:
```
Comment (bjin-cloudflare) [inline src/content/docs/china-network/concepts/china-dns.mdx:48]: Reviewed and suggested by [crwaters16](https://github.com/crwaters16)

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**